source/clear-linux/get-started/bootable-usb/bootable-usb.rst

```diff
@@ -1,32 +1,203 @@
 .. _bootable-usb:
 
-Create a bootable |CL-ATTR| USB drive
-#####################################
+Create a bootable USB drive
+###########################
 
 Instructions to create a |CL-ATTR| USB drive vary depending on your operating
-system.
+system. Follow the instructions applicable to your system:
 
-.. _download-usb-image:
+* :ref:`bootable-usb-linux`
+* :ref:`bootable-usb-mac`
+* :ref:`bootable-usb-windows`
 
-Download the latest |CL| image
-******************************
+Prerequisites
+*************
 
-There are 2 types of |CL| images suitable for burning onto and running
-off a USB drive:
+* Use a **16GB** or larger USB drive.
+* `Download`_ the |CL| live boot image or interactive installer image.
 
-* Live image: :file:`clear-[version number]-live.img.xz`
-* Installer image: :file:`clear-[version number]-installer.img.xz`
+.. _bootable-usb-linux:
 
-Go to the |CL| `image`_ repository and download the desired type.
+Create a bootable USB drive on Linux
+************************************
 
-With the appropriate image downloaded, choose the step-by-step instructions
-applicable to your system:
+Make sure you have have completed all `Prerequisites`_.
 
-.. toctree::
-    :maxdepth: 1
+Before burning the image onto your USB drive,
+:ref:`verify and decompress your image <verify-linux>`.
 
-    bootable-usb-linux
-    bootable-usb-windows
-    bootable-usb-mac
+Burn the |CL| image onto a USB drive
+====================================
 
-.. _image: https://download.clearlinux.org/image
+.. caution::
+
+   |CAUTION-BACKUP-USB|
```

Comment:
In the repeated CAUTION-BACKUP-USB text, the phrase "Backup your data" should be "Back up your data". 
(Similar usage as setup when it's a thing & set up when it's an action.)